hardware-api.md

```diff
@@ -128,6 +128,16 @@ i2c.transfer(new Buffer([0xde, 0xad, 0xbe, 0xef]), function (err, rx) {
 })
 ```
 
+### Terms Used
+
+The following sections use industry standard technical terms that are considered non-inclusive. They are used here in an explicitly technical manner and only to be accurate in describing these wire communication protocols. The terms are used strictly as defined here:
+
+**Master:** A machine or device directly controlling another (source: [Oxford Dictionary](http://www.oxforddictionaries.com/us/definition/american_english/master#master__7))
+
+**Slave:** A device, or part of one, directly controlled by another (source: [Oxford Dictionary](http://www.oxforddictionaries.com/us/definition/american_english/slave#slave__7))
+
+
+
```

Comment:
But that spacing isn't used anywhere else in the file? I think it's best to stay consistent and always space with one blank line...